librz/core/cmd_analysis.c

```diff
@@ -6042,6 +6042,73 @@ static void cmd_analysis_ucall_ref(RzCore *core, ut64 addr) {
 	}
 }
 
+static RzFlagItem *core_flag_get_at_as_ref_type(RzCore *core, RzAnalysisXRef *xrefi) {
+	RzListIter *iter;
+	RzFlagItem *item;
+	const RzFlagsAtOffset *flags_at = rz_flag_get_nearest_list(core->flags, xrefi->to, -1);
+#define is_not_in_space(f, i) (rz_flag_space_cur(f) && (i)->space != rz_flag_space_cur(f))
```

Comment:
Please separate define from the code with the newlines. Also it might be sensible to swap these two lines to avoid the "flow interruption"

Reply:
In fact, I think it's better to make this a small `inline` function instead of the define.

Reply:
i am changing this